browser/ui/brave_browser_command_controller_browsertest.cc

```diff
@@ -48,6 +77,16 @@ IN_PROC_BROWSER_TEST_F(BraveBrowserCommandControllerTest,
       command_controller->IsCommandEnabled(IDC_NEW_OFFTHERECORD_WINDOW_TOR));
 #endif
 
+#if BUILDFLAG(ENABLE_BRAVE_VPN)
```

Comment:
It would be nice to break this one giant method into 4 separate methods:
- BraveCommandsEnableTest
- BraveCommandsEnableTestPrivate
- BraveCommandsEnableTestGuest
- BraveCommandsEnableTestPrivateTor

Maybe I can add a commit to update this

Reply:
Fixed with https://github.com/brave/brave-core/pull/9916/commits/3c68aed34fc8e2bdee230cfcc05e8b48e01d8109 😄